### PR TITLE
Support loading module from command line 

### PR DESCRIPTION
**Pull request by yrajas**
Add support for --loadmodulecs argument to support loading of modules at startup time.

### CODE REVIEW

File: libs/server/Module/ModuleUtils.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+            // If necessary, check that all assemblies are digitally signed
+            if (!allowUnsignedAssemblies)
+            {
+                foreach (var loadedAssembly in loadedAssemblies)
+                {
+                    var publicKey = loadedAssembly.GetName().GetPublicKey();
+                    if (publicKey == null || publicKey.Length == 0)
+                    {
+                        errorMessage = CmdStrings.RESP_ERR_GENERIC_ASSEMBLY_NOT_SIGNED;
```
Comment (PaulusParssinen) [inline libs/server/Module/ModuleUtils.cs:59]: I wonder if there's any concerns about not unloading the assembly upon failure of signature verification? 🤔

Comment (yrajas) [reply libs/server/Module/ModuleUtils.cs:59]: Good point.

The locations will have to be allowed by the admin for the assembly to be loaded through the 'LOAD MODULECS' command or the entire assembly specified using the --loadmodulecs argument. So, there is definitely admin control on what gets loaded. 

If the module initializer invocation cannot be precisely controlled, wondering would even explicitly unloading give any better security guarantees.

Comment (PaulusParssinen) [reply libs/server/Module/ModuleUtils.cs:59]: Unloading wouldn't help in that case, you're correct. We could avoid loading the assembly for the check by using `System.Reflection.Metadata` to inspect the PE file manually:

 ```csharp
using var fs = File.OpenRead(modulePath);
using var peReader = new PEReader(fs);

var metadataReader = peReader.GetMetadataReader();
var assemblyPublicKeyHandle = metadataReader.GetAssemblyDefinition().PublicKey;

// if (assemblyPublicKeyHandle.IsNil) ..

var publicKeyBytes = metadataReader.GetBlobBytes(assemblyPublicKeyHandle);
```

But this also doesn't give any additional security guarantees AFAIK, as [.NET does not validate strong names at runtime](https://learn.microsoft.com/en-us/dotnet/standard/assembly/strong-named) anymore.

Comment (yrajas) [reply libs/server/Module/ModuleUtils.cs:66]: Agreed. 
Updated to avoid loading assembly if signing check fails as a defensive measure.

**Review by badrishc — APPROVED** [review]

**Pull request merged**